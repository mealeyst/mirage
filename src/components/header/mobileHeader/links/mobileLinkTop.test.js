import React from 'react'
import 'jest-styled-components'

import { MobileLinkTop } from 'SRC'

const { shallowWithTheme } = global

describe('(Styled Component) MobileLinkTop', () => {
  const createMobileLinkTop = (props) => {
    return shallowWithTheme(<MobileLinkTop {...props} />)
  }

  test('matching the snapshot', () => {
    expect(createMobileLinkTop())
    .toMatchSnapshot()
  })
})
