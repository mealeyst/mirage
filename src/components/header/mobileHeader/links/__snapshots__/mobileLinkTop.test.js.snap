// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Styled Component) MobileLinkTop matching the snapshot 1`] = `
.c0 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  height: 4rem;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  background-color: transparent;
  padding-left: 0.5rem;
  padding-right: 0.5rem;
}

.c0:active,
.c0:focus,
.c0:hover,
.c0:visited {
  color: #0073D1;
}

<Styled(Styled(Component))
  background="transparent"
  className="c0"
  fontFamily="\\"din-cond\\", \\"din-condensed-web\\", Arial, sans-serif"
  fontSize="2.6rem"
  underline={false}
  uppercase={true}
/>
`;
